docs/source/dev_guide/testing.rst

```diff
@@ -1,22 +1,60 @@
 Testing E3SM Diagnostics
 ==============================================
 
-Unit tests
-----------
+Unit and integration tests
+--------------------------
 
-Run all unit tests by doing the following:
+Run all automated tests by doing the following:
 
     .. code::
 
         pip install . # Install your changes
-        ./tests/test.sh # Run all unit tests
+        ./tests/test.sh # Run all unit and integration tests
+
+
+If these tests pass, you're done. If they fail unexpectedly however,
+your code might have changed what the output looks like.
+
+``ls tests/system/image_check_failures`` will show your all the images that differ from expected.
```

Comment:
Missed this. Need to change this path.

Reply:
(This should be `ls tests/integration/image_check_failures`)

Reply:
Should be "will show you "